keps/sig-storage/5030-attach-limit-autoscaler/kep.yaml

```
@@ -0,0 +1,50 @@
+title: Integrate CSI Volume attach limits with cluster autoscaler
+kep-number: 5030
+authors:
+  - "@gnufied"
+owning-sig: sig-storage
+participating-sigs:
+  - sig-storage
+  - sig-scheduling
+  - sig-autoscaling
+status: provisional
+creation-date: 2025-01-09
+reviewers:
+  - TBD
+  - "@jsafrane"
+  - "@msau42"
+approvers:
+  - TBD
+
+see-also:
+  - "/keps/sig-aaa/1234-we-heard-you-like-keps"
+  - "/keps/sig-bbb/2345-everyone-gets-a-kep"
+replaces:
+  - "/keps/sig-ccc/3456-replaced-kep"
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: alpha|beta|stable
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "v1.33"
```

to satisfy the requirement for the upcoming Enhancements Freeze.

```suggestion
latest-milestone: "v1.34"
```